releasenotes/notes/pilot-discovery-scoped-namespaces.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+# issue is a list of GitHub issues resolved in this note.
+# If issue is not in the current repo, specify its full URL instead.
+issue:
+  - https://github.com/istio/istio/issues/26679
+
+# releaseNotes is a markdown listing of any user facing changes. This will appear in the
+# release notes.
+releaseNotes:
+- |
+  **Added** An option for dynamically restricting the set of namespaces for Services, Pods, and Endpoints that istiod processes when pushing xDS updates to improve performance on the data plane.
```

Comment:
Can you specify the option (DiscoverySelectors?) here? 

As a follow up - Would love to have a some doc that teaches user how to use this.  The guideline is to put doc on istio wiki for experimental feature (I assume this is).

Reply:
also wonder if it is useful to ask users to visualize the mesh (like using kiali) to determine the correct configuration for the discovery selectors.

Reply:
I'm happy to follow up with documentation on how to utilize this feature, I'll follow up offline to discuss